lib/improver/utilities/cube_metadata.py

```diff
@@ -95,6 +95,43 @@ def update_stage_v110_metadata(cube):
     return True
 
 
+def update_units(item, changes):
+    """Update the units on the item supplied. Options are to either
+    'set' or 'convert' the units
+
+    Args:
+        item (iris.cube.Cube, iris.coords.DimCoord or iris.coords.AuxCoord):
+            Item that will have its units modified.
+        changes (dict):
+            Dictionary of the formats shown below for specifying whether the
+            units will be set or converted::
+            "units": {
+                "action": "set",
+                "units": "K"
+            }
+            "units": {
+                "action": "convert",
+                "units": "seconds since 1970-01-01 00:00:00"
+            }
+
+    Return:
+        item (iris.cube.Cube, iris.coords.DimCoord or iris.coords.AuxCoord):
+            Item that has had its units modified.
+    """
+
+    if changes["units"]["action"] == "set":
+        item.units = changes['units']["units"]
+    elif changes["units"]["action"] == "convert":
+        item.convert_units(changes["units"]["units"])
+    else:
+        msg = ("The specified action is not known. "
+               "The specified action was '{}'. "
+               "Possible actions are: 'set' or 'convert'.").format(
+                   changes["units"]["action"])
+        raise KeyError(msg)
+    return item
```

Comment:
I think we modify the item in place, so we don't need a return statement, but would be good to mention that in the docstring.

Reply:
This function has been removed.